spec/features/data_access/data_download_spec.rb

```diff
@@ -272,6 +272,34 @@
         end
       end
     end
+    context 'upon clicking a "Download Data Links File" button' do
+      before :all do
+        click_link "Download Data Links File"
+      end
+
+      it "displays a page with a shell script on it which performs the user's query" do
+        within_last_window do
+          expect(page).to have_content('ftp://ladsftp.nascom.nasa.gov/allData/5/MYD02QKM/2014/191/MYD02QKM.A2014191.0330.005.2014191162458.hdf')
+        end
+      end
+
+      context 'and click "Download Links File" button' do
+        before :all do
+          within_last_window do
+            synchronize do
+              expect(page).to have_link('Download Links File')
+            end
+            click_link 'Download Links File'
+          end
+        end
+
+        it 'downloads a list of links' do
+          within_last_window do
```

Comment:
Is this link beyond the first page? I think it be valuable to ensure that the loading of more than the first page

Reply:
@abbottry I looked up the collection and it appears to only have 15 granules, so no - no second page (2000 granules per page).  Further the collection seems to be another one that is 'gone' and can't be found live - seems to be a recording, only (I hate that).

I will look around for a larger collection to use - one that's maybe just a bit over 2000 would be nice.

Reply:
@abbottry As stated in an email I'm having some trouble finding a collection that is > 2000 granules that won't throw errors, since rspec does not seem to be able to login.  :(

Reply:
@abbottry test updated - I left the original test above, plus added one with a 2000+ granule collection.